src/main/main.ts

```diff
@@ -11,16 +11,33 @@
 import path from 'path';
 import { app, BrowserWindow, shell } from 'electron';
 import { autoUpdater } from 'electron-updater';
-import log from 'electron-log';
+import log, { LogMessage } from 'electron-log';
 import 'reflect-metadata';
 import './ipc';
+import chalk from 'chalk';
 import AppDataSource from '../data-source';
-import { resolveHtmlPath } from './util';
+import { resolveHtmlPath, getDataPath } from './util';
 import MenuBuilder from './menu';
 
 class AppUpdater {
   constructor() {
+    log.transports.file.resolvePath = () =>
```

Comment:
Should we add chalk to file logs?

Reply:
Chalk is just color formatting for console